src/Tasks/AssemblyDependency/AssemblyInformation.cs

```diff
@@ -361,6 +361,21 @@ private void CorePopulateMetadata()
                 using (var stream = File.OpenRead(_sourceFile))
                 using (var peFile = new PEReader(stream))
                 {
+                    bool hasMetadata = false;
+                    try
+                    {
+                        hasMetadata = peFile.HasMetadata;
```

Comment:
nit: carry the comment over that explains why we do this.

Reply:
I've copied over both comments.